docs-ref-conceptual/authenticate-azure-cli-interactively.md

```diff
@@ -14,7 +14,30 @@ keywords: az login, authentication types, authentication methods, azure, cli log
 
 # Sign in interactively with Azure CLI 
 
-The Azure CLI's default authentication method for logins uses a web browser and access token to sign in.
+Interactive logins to Azure offer a more intuitive and flexible user experience. Interactive login with Azure CLI allows users to authenticate to Azure directly through the [az login](/cli/azure/reference-index#az-login) command, which is useful for ad-hoc management tasks and for environments that require manual sign-in, such as those with multi-factor authentication (MFA). This method simplifies access for script testing, learning, and on-the-fly management without needing to pre-configure service principals or other noninteractive authentication methods.
+
+## Sign in with Web Account Manager (WAM) on Windows
+
+As of Azure CLI version [3.0](./release-notes-azure-cli.md#may-21-2024), WAM is now the default authentication method on Windows. WAM is a Windows 10+ component that acts as an authentication broker. (An authentication broker is an application that runs on a user’s machine that manages the authentication handshakes and token maintenance for connected accounts.)
```

Comment:
We won't bump to `3.0`.

Also, should we use `'` instead of `’`?

```suggestion
As of Azure CLI version [2.61.0](./release-notes-azure-cli.md#may-21-2024), WAM is now the default authentication method on Windows. WAM is a Windows 10+ component that acts as an authentication broker. (An authentication broker is an application that runs on a user's machine that manages the authentication handshakes and token maintenance for connected accounts.)
```